Description: Hardcode path to /usr/share/fonts/truetype/ttf-dejavu/DejaVuSans.ttf in imagick-3.1.0RC1/examples/captcha.php
 Hardcode-path-to-usrsharefontstruetypettf-dejavuDejaVuSans.ttf-in-captcha.php
Forwarded: not-needed
Author: Thomas Goirand <zigo@debian.org>
--- php-imagick-3.1.0~rc1.orig/imagick-3.1.0RC1/examples/captcha.php
+++ php-imagick-3.1.0~rc1/imagick-3.1.0RC1/examples/captcha.php
@@ -17,7 +17,7 @@ $bg->setColor( 'white' );
 $ImagickDraw = new ImagickDraw();
 
 /* Set font and font size. You can also specify /path/to/font.ttf */
-$ImagickDraw->setFont( 'Helvetica Regular' );
+$ImagickDraw->setFont( '/usr/share/fonts/truetype/ttf-dejavu/DejaVuSans.ttf' );
 $ImagickDraw->setFontSize( 20 );
 
 /* Create the text */
